backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -30,10 +30,16 @@ def admin_hint(title, text)
         end
       end
 
-      def datepicker_field_value(date)
-        unless date.blank?
-          l(date, format: t('spree.date_picker.format', default: '%Y/%m/%d'))
+      def datepicker_field_value(date, with_time = false)
```

Comment:
Can we use a named argument here?

```suggestion
      def datepicker_field_value(date, time: false)
```